datafusion/core/tests/sqllogictests/src/main.rs

```diff
@@ -84,7 +84,10 @@ async fn run_test_file(
     relative_path: PathBuf,
 ) -> Result<(), Box<dyn Error>> {
     info!("Running with DataFusion runner: {}", path.display());
-    let test_ctx = context_for_test_file(&relative_path).await;
+    let Some(test_ctx) = context_for_test_file(&relative_path).await else {
+        info!("Skipping: {}", path.display());
```

Comment:
Yes, it will log twice. I was thinking that I wanted to log the reason the test was skipped (in `context_for_test_file` ) as well as make double sure that the logs showed any test that was skipped (if/when we add more tests that are skipped)

Given that the logs aren't printed by default (you need to do `RUST_LOG=...` to do so ) I chose the "mo logs mo better" approach, but I can see how others might have a different preference

I don't feel strongly and will change it if others prefer less logging